Source/Infrastructure/Read/MongoDb/IHaveExtraElements.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Infrastructure.Read.MongoDb
+{
+    /// <summary>
+    /// Defines a Read Model class that has extra bson documents.
+    /// ExtraElements will be a dumping place for all extra bson elements when serialized
+    /// </summary>
+    public interface IHaveExtraElements
```

Comment:
Do you have examples for when this is bredde? Why not use explicit typed properties?

Reply:
ExtraElements is a dump-place for extra elements on the read model. If we have a readmodel with fields that are not present in each of the documents in the mongo collection the application crashes